Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -335,11 +340,17 @@ bool MainWindow::Stop()
                                            "may be lost if you stop the current emulation "
                                            "before it completes. Force stop?") :
                                         tr("Do you want to stop the current emulation?"));
+
+    m_inquiry_in_progress = false;
+
     if (pause)
       Core::SetState(state);
 
     if (confirm != QMessageBox::Yes)
+    {
+      m_exit_requested = false;
```

Comment:
This doesn't seem right to me. If I've already asked to close Dolphin and confirmed that I want to stop the emulation, then get impatient and click "close" again, declining to force stop the emulation shouldn't negate my previous desire to close the entire application.